projects/common/lib/components/replace-link-pipe/i18n/index.js

```diff
@@ -0,0 +1,4 @@
+module.exports = require.context(
```

Comment:
Is this file necessary? I think it can be removed and we can just use Typescript `imports` in the parent/children as necessary.